simconvoi.cc

```diff
@@ -3453,6 +3453,20 @@ bool convoi_t::pruefe_alle()
 	return ok;
 }
 
+/**
+ * Check sum power of all coupling convoys
+ * Only call this function by the front convoy!
+ */
```

Comment:
この説明はヘッダファイルに書いてください。
```suggestion
/**
 * Returns sum power of this and the child convoys.
 */
```